pages/Category.vue

```diff
@@ -309,7 +309,26 @@ export default {
         });
     },
     categories () {
-      return this.prepareCategories(this.getCategories[0].children_data);
+      return getTopLevelCategories(this.getCategories)
+        .map(category => {
+          const viewAllMenuItem = {
+            ...category,
+            name: i18n.t('View all'),
+            position: 0
+          };
+
+          return {
+            ...this.prepareCategoryMenuItem(category),
+            items: [this.prepareCategoryMenuItem(viewAllMenuItem)]
+              .concat(
+                category.children_data.map(childCategory => this.prepareCategoryMenuItem(
+                  this.getCategories.find(category => category.id === childCategory.id)
+                ))
```

Comment:
can you add here `.filter(Boolean)` to remove undefined from array? 
`prepareCategoryMenuItem` can return undefined

Reply:
Sure, done  👍